src/graph.rs

```diff
@@ -138,29 +138,66 @@ impl Range {
 }
 
 #[derive(Debug, Clone)]
-pub enum ModuleGraphError {
+pub enum ModuleError {
```

Comment:
By the way, we can't land this PR until probably next month in preparation for Deno 2.0. If you want to get some of these non-breaking changes into main faster then you might want to consider refactoring these out ot a separate PR and I'll take a look at it to get it merged quickly.